homeassistant/components/fan/device_action.py

```diff
@@ -14,13 +18,16 @@
 
 async def async_get_actions(
     hass: HomeAssistant, device_id: str
-) -> list[dict[str, str]]:
+) -> GetAutomationsResult:
```

Comment:
I would like to revert this, for reasoning and the review see: <https://github.com/home-assistant/core/pull/72163#issuecomment-1133865649>